apps/docs/src/.vitepress/components/DesignVariable.vue

```diff
@@ -97,10 +102,17 @@ const emit = defineEmits<{
     &--no-value {
       padding: var(--onyx-spacing-3xs) var(--onyx-spacing-2xs);
     }
+    &--no-padding {
+      padding: 0;
+    }
   }
 
   &__value {
+    color: var(--onyx-color-base-neutral-500);
```

Comment:
```suggestion
    color: var(--onyx-color-text-icons-neutral-soft);
```

Variable as used in Figma :) Has better dark mode contrast